package.json

```diff
@@ -29,6 +29,10 @@
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.10.4",
     "@emotion/babel-preset-css-prop": "^10.0.27",
+    "@guardian/src-button": "2.4.0",
+    "@guardian/src-foundations": "2.4.0",
+    "@guardian/src-grid": "2.4.0",
+    "@guardian/src-icons": "2.4.0",
```

Comment:
Ah so these need to be duplicated here because we need them for development. I'd been experimenting with using https://www.npmjs.com/package/install-peers-cli to install peer deps for local dev. Any thoughts on one approach or the other?

Reply:
I think using `devDependencies` is fine, using `install-peers-cli` adds an extra layer that may not necessarily be obvious to new devs. But as long as it is clearly communicated in the ReadMe then I think its fine.